### ISSUE TITLE
Apparent bug in read_wac function

### ISSUE THREAD

**Issue by rhedley** [WavX/bioacoustics#12]
Hi there, 
I have run into an error when using the read_wac function. To summarize, Wildlife Acoustics has two naming conventions for files. The standard naming convention is prefix_date_time.wac, and the convention if a GPS is attached and has a location fix is prefix_date$time.wac

read_wac seems to work fine with the standard naming convention, but when I run it on a file with the GPS naming convention, I get the error 
"Error in read_wac("P:/BG/2015/01/BG-0004/BG-0004-040/BG-0004-040_20150618$050000.wac") : 
  Could not extract date/time info from filename: 'P:/BG/2015/01/BG-0004/BG-0004-040/BG-0004-040_20150618$050000.wac'"

I have looked at the code for the function, and figured out that this seems to come from these lines of code:

```
date_time <- strptime(substr(fpse, nchar(fpse) - 14, nchar(fpse)), 
    "%Y%m%d_%H%M%S")
  if (is.na(date_time)) 
    stop("Could not extract date/time info from filename: '", 
      file, "'")
```

Basically, it is looking for the _ in the name, and finding a $

I haven't looked into this too much, but I think a simple fix would be to add this to the code snippet above:
```

date_time <- substr(fpse, nchar(fpse) - 14, nchar(fpse))
date_time <- gsub(date_time,pattern='\\$', replacement='_')
date_time <- strptime(date_time, 
    "%Y%m%d_%H%M%S")

```
This could replace the original one line of code defining date_time.
That's one way I think it could be fixed, I'm sure there are other ways you might prefer... Just thought I'd bring it to your attention.

**Comment by ygc2l** [comment]
Thanks for reporting ! Fixed in 14bbe76.
